app/views/physical_server/_form.html.haml

```diff
@@ -0,0 +1,108 @@
+- @angular_form = true
```

Comment:
This seems to be a copy-paste of the `host` angular form.

To simplify things a little bit, I recommend removing all angular related code from this PR.
It would be a good idea to work on the angular bits after this gets merged.